docs/source/getting_started.rst

```diff
@@ -10,7 +10,34 @@ Start by importing FURY.
     from fury import window, actor, ui, io, utils
 
 To import a model, use :py:func:`.io.load_polydata`. Currently supported formats include OBJ, VTK, FIB, PLY, STL and XML.
-Let us include the ``suzanne`` model used by Blender
+
+Here we are going to use ``suzanne`` model used by Blender. You can find more detail about what is suzanne model `here <https://en.wikipedia.org/wiki/Blender_(software)#:~:text=A%20low-polygon%20model%20with%20only%20500%20faces%2C%20Suzanne,gives%20out%20an%20award%20called%20the%20Suzanne%20Award.).>`_.
+
+We can download the model by
+
+.. code-block:: python
+
+    from fury.data import fetch_viz_models
+    fetch_viz_models()
+    
+Then the models will be downloaded in ``~/.fury/models``
+
+If store the return value to ``models``, you can find the detailed information in the dictionary. The directory of the ``suzanne`` model will be in ``model[1]``.
```

Comment:
typo: if ?

typo:  models[1].    `s` missing to be coherent with the code below

Reply:
love the process of polishing:) done👍